agenta-web/src/components/Evaluations/EvaluationErrorModal.tsx

```diff
@@ -1,15 +1,30 @@
-import React from "react"
+import React, {Dispatch, SetStateAction} from "react"
 import {Modal, Button} from "antd"
```

Comment:
I can't find any docs about Dispatch and SetStateAction

Reply:
@aakrem it's a ts type for react. See https://github.com/typescript-cheatsheets/react/blob/f0bd7c5258b69698aa99ea16cc3f08af05cd3ba7/docs/basic/getting-started/basic-type-examples.md

Reply:
I think we just need to use setState it looks weird for me.
@MohammedMaaz can you check here?

Reply:
I think SetStateAction is the type. We already use it in many places in the code (https://github.com/search?q=repo%3AAgenta-AI%2Fagenta%20SetStateAction&type=code). If I remember correctly it was introduced by Suad